OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -133,6 +134,9 @@
     @Inject
     private VideoDownloadHelper downloadManager;
 
+    @Inject
```

Comment:
there is no need to add `@Inject` as initializing at L#182.

Reply:
Removed